src/components/util/search-matching-equipments.js

```diff
@@ -37,15 +36,14 @@ export const useSearchMatchingEquipments = (
                         studyUuid,
                         nodeUuid,
                         searchTerm,
-                        useNameLocal,
+                        getParameter,
                         inUpstreamBuiltParentNode,
                         equipmentType
                     )
                         .then((infos) => {
                             if (searchTerm === lastSearchTermRef.current) {
-                                setEquipmentsFound(
-                                    makeItems(infos, useNameLocal)
-                                );
+                                const label = getNameOrId();
```

Comment:
you means `getParameter()` here I think no ?

Reply:
`setEquipmentsFound(makeItems(infos, getUseNameParameterKey()));`

Reply:
I changed in make Items so we must pass the label directly

Reply:
My bad, but it still doesn't work 
calling getNameOrId() without any parameter will always return null now
Here you want to pass the function `getNameOrId` to makeItems aka getEquipmentsInfosForSearchBar then 
`setEquipmentsFound(makeItems(infos, getNameOrId));`
and
```
var getEquipmentsInfosForSearchBar = function getEquipmentsInfosForSearchBar(equipmentsInfos, equipmentLabelling) {
  return equipmentsInfos.flatMap(function (e) {
    var label = equipmentLabelling(e);
    return e.type === 'SUBSTATION' ? [{
    ...
        type: e.type,
        voltageLevelLabel: equipmentLabelling(vli),
        voltageLevelId: vli.id
```